### ISSUE TITLE
Provide "default" Ajv as a named export

### ISSUE THREAD

**Issue by JacobLey** [ajv-validator/ajv#1518]
**What version of Ajv you are you using?**
7.2.4

**What problem do you want to solve?**

tldr: named + default exports are not simultaneously possible with CJS. Pick one (probably named).

The default export of Ajv is the Ajv class https://github.com/ajv-validator/ajv/blob/master/lib/ajv.ts#L37

which implies equivalent ESM code like `import Ajv from 'ajv'`.

That works in babel/typecript environments that transform the code into something like `const Ajv = require('ajv').default`

But with NodeJS's native ESM, default export/import works a bit differently: https://nodejs.org/api/esm.html#esm_import_statements

The "default" export is actually module.exports, and accessing the intended "default" export requires:
```
import DefaultAjv from 'ajv';
const Ajv = DefaultAjv.default;
```
At best that is ugly, at worst it is actively broken when writing the code with Typescript+ESM.

This can be checked by running `import('ajv').then(console.log)` and notice the output is something like `{ default: { default: Ajv } }`

**What do you think is the correct solution to problem?**
The simplest solution is probably to provide Ajv as a named export.
e.g. add `export { Ajv }` to the main file.

That allows pure ESM solutions like `import { Ajv } from 'ajv'`, and this would be non-breaking.

_Alternatively_ it can use a correct default export, and export the Ajv class as `module.exports = Ajv`. This would mean any other named exports have to be tacked onto the Ajv class rather than exported in parallel.

**Will you be able to implement it?**
Happy to open a PR and update docs to match, just wanted to expose reasoning as well.

**Comment by epoberezkin** [comment]
It’s resolved in v8 - it’s worth migrating to it, it is mostly compatible.

**Comment by JacobLey** [comment]
Oh yes it does appear to be fixed!
Great!

In my defense... it wasn't released 30 minutes ago 😉

**Comment by leefsmp** [comment]
Using 8.0.2 in typescript and this isn't compiling:

```
import ajv from "ajv"
ajv.compile() //compile not defined on ajv
```

this is also not working:

```
import DefaultAjv from 'ajv';
const Ajv = DefaultAjv.default;
```

What am I missing? Thanks

**Comment by JacobLey** [comment]
You need to call `new Ajv()` first

**Comment by epoberezkin** [comment]
See the examples in Getting started

**Comment by paul-uz** [comment]
@epoberezkin the Get Started examples are wrong. They do not show valid code for the latest version of AJV (v8).

Importing using `const Ajv = require('ajv');` results an error when calling `const ajv = new Ajv();`, "This expression is not constructable"

None of the documentation mentions that the library should be imported using `const Ajv = require('ajv').default;`
